### PR TITLE
Add the section how to build the browser

### PR DESCRIPTION
**Pull request by autn**
Hi @YPCrumble, this PR is used for adding the section on how to build the browser.
Please help me to review.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -19,3 +19,13 @@ Nightly development builds can be downloaded from:
 
 Keep in mind that nightly builds, which are used by Firefox developers for
 testing, may be buggy.
+
+Setup:
+
+    - Clone this repository
+    - Add `mozilla-unified` repository: `git remote add hg hg::https://hg.mozilla.org/mozilla-unified`
```
[the remark this answers is not in the document]
Comment (autn) [reply README.txt:26]: @YBCrumble I get the errors when running build without pulling code from mozilla-unified. I followed some docs and it's solved by adding mozilla-unified repo. No change on git when running git pull hg.

[the remark this answers is not in the document]
Comment (autn) [reply README.txt:26]: @YPCrumble you can buildt for Mac, so would you please add the section for how to install on Mac? And we will change this section for Linux only?

**Comment by autn** [comment]
@YPCrumble what was the type you chosed?

**Comment by YPCrumble** [comment]
I’m pretty sure I chose number two - default Firefox style build

**Pull request merged**